src/component/AnimeList.jsx

```diff
@@ -0,0 +1,27 @@
+const AnimeList = ({ apiData }) => {
+  if (apiData.status === 404) {
+    return <h1>{apiData.message}</h1>;
+  } else {
+    const { results } = apiData;
```

Comment:
Instead of passing `apiData` props can we pass list directly?

Reply:
Implemented.